src/screens/Game/GameRequirements/index.tsx

```diff
@@ -0,0 +1,63 @@
+import React, { Fragment } from 'react'
```

Comment:
this is mainly the code extracted from GamePage and the table is not inside an InfoBox, but code is pretty much the same

Reply:
Good.
We need to split those big components like the GamePage and the Settings one.
So that was a good move.